exchanges/btse/btse_test.go

```diff
@@ -752,6 +752,30 @@ func TestStripExponent(t *testing.T) {
 	assert.ErrorIs(t, err, errInvalidPairSymbol, "Should error on a symbol with too many underscores")
 }
 
+func TestMarketPair(t *testing.T) {
+	t.Parallel()
+
+	for _, test := range []struct {
```

Comment:
Aside: I often call these `tt`. Here `test` caught me because I'd kinda expect that to be a pkg, and I can't recall seeing this naming.
```suggestion
	for _, tt := range []struct {
```

Don't have to action. Just in case.